.github/workflows/examples-mpi.yml

```diff
@@ -47,10 +47,13 @@ jobs:
 
     - name: Test mpi notebooks
       run : |
-        # Currently not tested due to issue #859
-        # ipcluster start --profile=mpi -n 4 --daemon
-        # py.test --nbval examples/mpi
-        # ipcluster stop --profile=mpi
+        pip3 install "ipyparallel<8.4"
```

Comment:
@georgebisbas out of curiosity, why <8.4 ? and shouldn't this rather be added to the "Install dependencies" section above?